libs/auth/src/artifact_authentication.ts

```diff
@@ -320,9 +320,11 @@ async function performArtifactSpecificAuthenticationChecks(
  */
 export async function prepareSignatureFile(
   artifact: ArtifactToExport,
-  /* istanbul ignore next - @preserve */
-  config: ArtifactAuthenticationConfig = constructArtifactAuthenticationConfig()
+  configOverride?: ArtifactAuthenticationConfig
 ): Promise<{ fileContents: Buffer; fileName: string }> {
+  const config =
+    configOverride ??
+    /* istanbul ignore next - @preserve */ constructArtifactAuthenticationConfig();
```

Comment:
Our latest test setup (Vitest 3) doesn't seem to register istanbul ignores of default param assignments. Rearranged the code to work around that